scripts/inc.installer.sh

```diff
@@ -1106,12 +1107,21 @@ ui_print " "
 # For reference, check https://github.com/osm0sis/AnyKernel3/blob/master/META-INF/com/google/android/update-binary
 ui_print "- Mounting partitions"
 set_progress 0.01
+
+ps | grep zygote | grep -v grep >/dev/null && BOOTMODE=true || BOOTMODE=false
+$BOOTMODE || ps -A 2>/dev/null | grep zygote | grep -v grep >/dev/null && BOOTMODE=true
```

Comment:
Maybe we can use getprop here instead using
[init.svc.zygote]: [running]

Reply:
This is the established script for $BOOTMODE, established by Magisk, and is known working in all cases, so I'd rather not mess with it.